packages/paste-core/components/anchor/src/index.tsx

```diff
@@ -26,43 +26,26 @@ export const secureExternalLink = (href: string): {} | undefined => {
   };
 };
 
-const Anchor = React.forwardRef<HTMLAnchorElement, AnchorProps>((props, ref) => (
-  <Text
-    as="a"
-    color="colorTextLink"
-    textDecoration="underline"
-    outline="none"
-    ref={ref}
-    _active={{
-      boxShadow: 'shadowFocus',
-      color: 'colorTextLinkDarker',
-      textDecoration: 'none',
-    }}
-    _focus={{
-      boxShadow: 'shadowFocus',
-      color: 'colorTextLinkDarker',
-      textDecoration: 'none',
-    }}
-    _hover={{
-      color: 'colorTextLinkDarker',
-      textDecoration: 'none',
-    }}
-    {...secureExternalLink(props.href)}
-    {...safelySpreadTextProps(props)}
-  >
-    {props.children}
-  </Text>
-));
+const Anchor = React.forwardRef<HTMLAnchorElement, AnchorProps>((props, ref) => {
+  const AnchorComponent = getAnchorComponent(props.variant || 'default');
```

Comment:
I know we started doing variant styles as maps, but should we do the same for variants themselves too?

```jsx
const AnchorVariants = {
  'inverse': InverseAnchor,
  'default': DefaultAnchor,
}

const Anchor = React.forwardRef<HTMLAnchorElement, AnchorProps>((({variant='default', ...props}), ref) => {
  const AnchorComponent = AnchorVariants[variant];
...
```

Reply:
@SiTaggart Does that help with performance, by getting rid fo the `getAnchorComponent` function?

Reply:
Same theory I guess, one less function and switch statement to run

Reply:
Looks like I may get into a dependency cycle issue because AnchorVariants relies on InverseAnchor and DefaultAnchor, which rely on AnchorProps (which AnchorVariants is a part of).

Reply:
Just pushed changes. Maybe what I have now works.